chain-impl-mockchain/src/ledger.rs

```diff
@@ -581,9 +584,17 @@ fn internal_apply_transaction(
     witnesses: &[Witness],
     fee: Value,
 ) -> Result<Ledger, Error> {
-    assert!(inputs.len() < 255);
-    assert!(outputs.len() < 255);
-    assert!(witnesses.len() < 255);
+    if inputs.len() > 256 {
```

Comment:
Please define these magic numbers as constants somewhere visible at the beginning of the module.